charts/gitops-operator/README.md

```diff
@@ -0,0 +1,48 @@
+# ⚓️ GitOps Operator Helm Deploy
+
+The GitOps  Helm Chart customizes and deploys the [RedHat GitOps Operator](https://github.com/redhat-developer/gitops-operator) written by Red Hat.
+
+## Installing the chart
+
+To install the chart from source:
+```bash
+# within this directory 
```

Comment:
I think it would be good for the commands to assume the user is in the root of the repository